pkg/oc/cli/admin/admin.go

```diff
@@ -84,6 +85,7 @@ func NewCommandAdmin(name, fullName string, f kcmdutil.Factory, streams genericc
 				cmdutil.ReplaceCommandName("kubectl", fullName, kubecmd.NewCmdDrain(f, streams)),
 				cmdutil.ReplaceCommandName("kubectl", fullName, ktemplates.Normalize(kubecmd.NewCmdTaint(f, streams))),
 				network.NewCmdPodNetwork(network.PodNetworkCommandName, fullName+" "+network.PodNetworkCommandName, f, streams),
+				logs.NewCmdLogs(logs.LogsRecommendedCommandName, fullName+" "+logs.LogsRecommendedCommandName, f, streams),
```

Comment:
not a specific request to change here, but since we stopped renaming commands I don't see a lot of value to the call structure like this anymore.

Reply:
I agree